iree/tools/BUILD

```diff
@@ -243,6 +243,7 @@ iree_cc_binary(
         "@com_google_absl//absl/types:span",
         "//iree/base:api",
         "//iree/base:api_util",
+        "//iree/base:localfile",
```

Comment:
Should technically add to https://github.com/google/iree/blob/main/iree/tools/CMakeLists.txt for consistency, even though it [does nothing](https://github.com/google/iree/blob/36abdfde40058e0a89f9b296840922e1650d2fb9/iree/base/internal/CMakeLists.txt#L97-L101).